lib/representer.ex

```diff
@@ -103,53 +248,61 @@ defmodule Representer do
     {node, represented}
   end
 
+  # module attributes
+  @reserved_attributes ~w(after_compile before_compile behaviour impl compile deprecated doc typedoc dialyzer file moduledoc on_definition vsn derive enforce_keys optional_callbacks)a
```

Comment:
I'm comparing with https://hexdocs.pm/elixir/Module.html and I'm missing some:
```suggestion
  @reserved_attributes ~w(after_compile before_compile behaviour impl compile deprecated doc typedoc dialyzer external_resource file moduledoc on_definition on_load vsn derive enforce_keys optional_callbacks)a
```

Reply:
Good catch, thank you for the thorough check 👍